python/ql/test/experimental/CWE-643/XpathLibTests/xpathSinks.ql

```diff
@@ -0,0 +1,6 @@
+import python
+import semmle.python.security.injection.Xpath
```

Comment:
If I move all files to experimental, I won't be able to include `Xpath.qll` which would make this test fail during compilation. How do I fix this?

Reply:
Just move your new Xpath module to `experimental/semmle/python/security/injection/Xpath.qll`, then you can import it with `import experimental.semmle.python.security.injection.Xpath` :blush: